lava-v2-jobs-from-api.py

```diff
@@ -45,6 +45,8 @@
 NFSROOTFS_URL = '/'.join([ROOTFS_URL, '{}', 'rootfs.tar.xz'])
 KSELFTEST_INITRD_URL = '/'.join([ROOTFS_URL, '{}', 'tests', 'rootfs.cpio.gz'])
 
+TESTS_ROOTFS_URL = 'http://staging-storage.kernelci.org/images/rootfs/debian/stretchtests/20180601.3/'
```

Comment:
This should be on `storage.kernelci.org`, not `staging-storage`.

Is the job to build and publish this file system all merged and deployed now?

Reply:
No, the jobs to build and publish debian images are not still deployed in api/storage.kci.org
This is blocked by https://github.com/kernelci/kernelci-build-staging/pull/35
I'm waiting for builder10 to be back to test a PR for that.